odnoklassniki-android-sdk/src/main/java/ru/ok/android/sdk/Shared.java

```diff
@@ -3,46 +3,47 @@
 import java.util.Arrays;
 import java.util.List;
 
-final class Shared {
-    static final String REMOTE_PORTAL = "https://ok.ru/";
-    static final String REMOTE_API = "http://api.ok.ru/";
-    static final String REMOTE_WIDGETS = "https://connect.ok.ru/";
+public final class Shared {
```

Comment:
The class was not intented for using in a client. What's the point of exposing it rather than making separated enum with activity result codes (like done in fb, for example)?